apps/explorer/test/support/factory.ex

```diff
@@ -813,6 +813,22 @@ defmodule Explorer.Factory do
       block_hash: block.hash,
       to_address_hash: address.hash,
       value: Enum.random(1..100_000),
+      value_from_fees: Enum.random(1..100_000),
+      value_from_mainchain: Enum.random(1..100_000),
+      index: fee_payment_index()
+    }
+  end
+
+  def fee_payment_no_mainchain_reward_factory do
+    block = insert(:block)
+    address = insert(:address)
+
+    %FeePayment{
+      block_number: block.number,
+      block_hash: block.hash,
+      to_address_hash: address.hash,
+      value: Enum.random(1..100_000),
+      value_from_fees: Enum.random(1..100_000),
```

Comment:
The purpose of the test should be for fee payments from older blocks. 
I checked w Paolo that `value_from_mainchain` and `value_from_fees` will not be present for older blocks.

So the method can be called something like `old_fee_payment_factory` and it should not have `value_from_fees` either.

Reply:
Apply this for the other tests too.

Reply:
I add a new factory method to manage the fee payment old format without the fields `value_from_mainchain` and `value_from fees`, this is used in the `fee_payment_controller_test` module to test both new and old formats.
Moreover I modified the factory method used in the `address_controller_test` and `block_controller_test` modules to have the `value_from_mainchain` present but set to zero to better simulate the real situation and check that the query param correctly filter the fields with `value_from_mainchain` not present or present and greater than zero.